zio-config/src/main/scala/zio/config/ProductBuilder.scala

```diff
@@ -1,18 +1,13 @@
 package zio.config
 
-import scala.language.implicitConversions
+import ProductBuilder._
 
 private[config] trait ProductBuilder[A, B] {
-  // TODO; discuss - this exists just to make user's life more easier.
-  // Inspired from argonaut's choice on codecjson in similar situation.
-  // https://github.com/argonaut-io/argonaut/blob/master/project/Boilerplate.scala#L275
-  implicit def unsafeGet1[X, Y](f: Option[Y]): Y = f.get
```

Comment:
And we finally got rid of the implicit as well.
The main issue with this usage of `.get`. It will never occur, but having `.get` in a library can raise concerns